app/controllers/v0/sign_in_controller.rb

```diff
@@ -320,19 +317,17 @@ def delete_cookies
 
     def auth_service(type)
       case type
+      when 'mocked_authentication'
```

Comment:
You'll have to double check with frontend but I believe the type they are sending is `vamock` 
```suggestion
      when 'vamock'
```